internal/pkg/vm/internal/errors/vminternal.go

```diff
@@ -3,28 +3,8 @@
 // This contents can be slowly placed on the vm internal.
 package internal
 
-import (
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/errors"
-)
-
-// TODO fritz require actors to define their exit codes and associate
```

Comment:
@phritz we finally knocked this one out!

Reply:
:)